### PR TITLE
Tests' re-org

### PR DESCRIPTION
**Pull request by oandreeva-nv**
This PR:

1. Moves multi-lora tests to multi-gpu folder, since it requires at least 2 A100s
2. Removes frequent loading and unloading of "vllm_opt" model from vllm_backend tests, which was causing issues with vllm's memory profiler and resulting in `tritonclient.utils.InferenceServerException: [StatusCode.INVALID_ARGUMENT] load failed for model 'vllm_opt': version 1 is at UNAVAILABLE state: Internal: AssertionError: Error in memory profiling. This happens when the GPU memory was not properly cleaned up before initializing the vLLM instance.`

### CODE REVIEW

**Comment by tanmayv25** [comment]
> Removes frequent loading and unloading of "vllm_opt" model from vllm_backend tests, which was causing issues with vllm's memory profiler and resulting in tritonclient.utils.InferenceServerException: [StatusCode.INVALID_ARGUMENT] load failed for model 'vllm_opt': version 1 is at UNAVAILABLE state: Internal: AssertionError: Error in memory profiling. This happens when the GPU memory was not properly cleaned up before initializing the vLLM instance.

Is this a bug which we need to raise with vLLM?

**Comment by oandreeva-nv** [comment]
Yes, I would need to prepare a reproducer, though.
________________________________

**Comment by tanmayv25** [comment]
> Removes frequent loading and unloading of "vllm_opt" model from vllm_backend tests, which was causing issues with vllm's memory profiler and resulting in tritonclient.utils.InferenceServerException: [StatusCode.INVALID_ARGUMENT] load failed for model 'vllm_opt': version 1 is at UNAVAILABLE state: Internal: AssertionError: Error in memory profiling. This happens when the GPU memory was not properly cleaned up before initializing the vLLM instance.

Should we open a bug with vLLM team?

**Comment by oandreeva-nv** [comment]
@tanmayv25 Yes, I would need to prepare a reproducer, though.

File: ci/L0_multi_gpu/vllm_backend/test.sh [hunk 1]
```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Copyright 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```
[the remark this answers is not in the document]
Comment (oandreeva-nv) [reply ci/L0_multi_gpu/vllm_backend/test.sh:2]: Good catch!

File: ci/L0_backend_vllm/vllm_backend/test.sh [hunk 2]
```diff
@@ -50,6 +50,8 @@ function assert_curl_success {
 
 rm -rf models && mkdir -p models
 cp -r ${SAMPLE_MODELS_REPO}/vllm_model models/vllm_opt
+sed -i 's/"gpu_memory_utilization": 0.5/"gpu_memory_utilization": 0.4/' models/vllm_opt/1/model.json
```
[the remark this answers is not in the document]
Comment (oandreeva-nv) [reply ci/L0_backend_vllm/vllm_backend/test.sh:58]: Added comment

File: ci/L0_multi_gpu/vllm_backend/test.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (oandreeva-nv) [reply ci/L0_multi_gpu/vllm_backend/test.sh]: revised copyright

**Review by tanmayv25 — APPROVED** [review]

**Pull request merged**